homework3/hw3/EventsViewController.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class EventsViewController: UIViewController, BaseViewProtocol {
+    private let topBackgroundView = UIView()
+    private let menuView = MenuView()
+    private let matchesTableView: UITableView = .init()
+    var sections: [LeagueSection] = []
```

Comment:
Ovo treba u ViewModel. Sve što nije direktno vezano view, treba biti tamo.

U ViewModelu trebaju biti network pozivi, pozivi u bazu, kalkulacije, sve što je potrebno za setiranje dataSourca koji view treba za prikaz.

Reply:
@tinajuresko ovo ipak nisi stavila u ViewModel?